server/src/main/java/io/deephaven/server/plugin/PluginModule.java

```diff
@@ -0,0 +1,8 @@
+package io.deephaven.server.plugin;
+
+import dagger.Module;
+
+@Module(includes = {ObjectTypesModule.class})
+public interface PluginModule {
```

Comment:
I've got a follow up PR for an application plugin; instead of refactoring the structure in the application plugin, I'm providing the follow-up structure for it here.